games-board/chessx/chessx-1.4.6.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A free and open source chess database application"
+DESCRIPTION="With ChessX you can operate your collection of chess games \
+in many ways: browse, edit, add, organize, analyze, etc."
+HOMEPAGE="https://chessx.sourceforge.net/"
+COPYRIGHT="2005 Michal Rudolf"
+LICENSE="GNU GPL v2"
+REVISION='1'
+SOURCE_URI="https://downloads.sourceforge.net/chessx/chessx/$portVersion/chessx-$portVersion.tgz"
```

Comment:
You can use SourceForge's redirector service:
```diff
-SOURCE_URI="https://downloads.sourceforge.net/chessx/chessx/$portVersion/chessx-$portVersion.tgz"
+SOURCE_URI="https://downloads.sourceforge.net/chessx/chessx-$portVersion.tgz"
```
This is handy because it automatically finds the right subdirectory for you. So if the owner moves the archive to another directory, SF will find it and serve it, after an automatic redirect.